spec/impl_guide.md

```diff
@@ -78,4 +78,200 @@ A `did:webs` could be moved to use another DID method that uses the AID for uniq
 [[def: KERI event stream chain of custody]]
 ~ This section is informative.
-The [[ref: KERI event stream]] represents a cryptographic chain of custody from the [[ref: AID]] itself down to the current set of keys and the cryptographic commitment to the next rotation key(s). The [[ref: KERI event stream]] also contains events that do not alter the [[ref: AID]] key state, but are useful for the DID document, such as the supported [[ref: hosts]], current set of service endpoints, etc. A did:webs resolver produces the DID document by processing the [[ref: KERI event stream]] to determine the current key state. We detail the different events in "Basic KERI event details" and show how they change the DID Document. The mapping from [[ref: KERI event stream]] to the properties of the DID Document is the core of the did:webs resolver logic.  Understanding the optimal way to update and maintain the [[ref: KERI event stream]] (publish static keri.cesr files, dyanamically generate the keri.cesr resource, etc) is beyond the scope of the spec, but a reference implementation of the resolver that details these techniques is being developed alongside this spec. The important concepts are that the entire [[ref: KERI event stream]] is used to produce and verify the DID document.
\ No newline at end of file
+The [[ref: KERI event stream]] represents a cryptographic chain of custody from the [[ref: AID]] itself down to the current set of keys and the cryptographic commitment to the next rotation key(s). The [[ref: KERI event stream]] also contains events that do not alter the [[ref: AID]] key state, but are useful for the DID document, such as the supported [[ref: hosts]], current set of service endpoints, etc. A did:webs resolver produces the DID document by processing the [[ref: KERI event stream]] to determine the current key state. We detail the different events in "Basic KERI event details" and show how they change the DID Document. The mapping from [[ref: KERI event stream]] to the properties of the DID Document is the core of the did:webs resolver logic.  Understanding the optimal way to update and maintain the [[ref: KERI event stream]] (publish static keri.cesr files, dyanamically generate the keri.cesr resource, etc) is beyond the scope of the spec, but a [[ref: did:webs Reference Implementation]] of the resolver that details these techniques is being developed alongside this spec. The important concepts are that the entire [[ref: KERI event stream]] is used to produce and verify the DID document.
+
+### Verifiable data on a TEL
+[[def: verifiable data on a TEL]]
+~ This section is informative.
+Below is an example that highlights how verifiable data is anchored to a [[ref: KEL]] using a [[ref: TEL]]. We use the spec [[ref: designated aliases]] feature as a real-world example. You can walk through this real-world example in the [[ref: did:webs Reference Implementation]].
+In that section we showed the attestation (self-issued credential) as follows:
```

Comment:
In that section?  Did you mean in the example in the reference implementation?

Reply:
```suggestion
The attestation (self-issued credential) is as follows:
```